dicodile/utils/dictionary.py

```diff
@@ -136,14 +136,30 @@ def compute_norm_atoms(D):
     return norm_atoms[:, 0]
 
 
+def compute_norm_atoms_from_DtD(DtD, n_atoms, atom_support):
+    t0 = np.array(atom_support) - 1
+    norm_atoms = np.array([DtD[(k, k, *t0)] for k in range(n_atoms)])
+    return norm_atoms.reshape(*norm_atoms.shape, *[1 for _ in atom_support])
```

Comment:
@tomMoral quick question, I wonder if the `reshape` makes sense within the function or whether it should be the caller's responsibility? (to me it would make sense that `compute_norm_atoms_from_DtD` would return a 1-dimensional array?)

Reply:
I think we should not reshape indeed. the `compute_norm_atoms` returns a vector I think

Reply:
I think `compute_norm_atoms` extends dimensions as well? (e.g in `test_dicodile`, with 2 1-dimensional atoms, `compute_norm_atoms` returns an array of shape (2, 1) and not (2, )

Anyway I'll move the reshape to the callers